examples/single-page-app/docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+services:
+  ui:
+    build:
+      context: ../shared/node
+      target: yarn-routed
+    volumes:
+    - .:/workspace
+    working_dir: /workspace/ui
+
+  envoy:
+    depends_on:
+      ui:
+        condition: service_healthy
+      myhub:
+        condition: service_started
+      myhub-api:
+        condition: service_started
+    build:
+      context: .
+      dockerfile: ../shared/envoy/Dockerfile
+      args:
+        ENVOY_CONFIG: ${ENVOY_CONFIG:-envoy.yml}
+    ports:
+    - ${PORT_DEV_PROXY:-10001}:10001
+    - ${PORT_PROXY:-10000}:10000
+    command:
+      - envoy
+      - "-c"
+      - /etc/envoy.yaml
+      - "--component-log-level"
+      - oauth2:trace  # ,http:debug
+    entrypoint:
+      - /bin/bash
+      - -c
+      - |
+        set -eo pipefail
+        touch /var/lib/envoy/lds.yml
+        chmod +w /var/lib/envoy/lds.yml
```

Comment:
Why is configuration writable inside the Docker image after startup?

Reply:
its written

Reply:
for ref its written here https://github.com/envoyproxy/envoy/blob/afef2cadc493e6638134584806b231064499fda4/examples/shared/node/scripts/build-routed.sh#L1-L11